daemons/ipa-sam/ipa_sam.c

```diff
@@ -2728,7 +2728,7 @@ static NTSTATUS get_trust_pwd(TALLOC_CTX *mem_ctx, const DATA_BLOB *auth_blob,
 	if (iopw.count != 0 && iopw.current.count != 0 &&
 	    iopw.current.array[0].AuthType == TRUST_AUTH_TYPE_CLEAR) {
 		if (pwd != NULL) {
-			if (!convert_string_talloc(tmp_ctx, CH_UTF16, CH_UNIX,
+			if (!convert_string_talloc(tmp_ctx, CH_UTF16LE, CH_UNIX,
```

Comment:
CH_UTF16 and CH_UTF16LE are the same:
```
typedef enum {CH_UTF16LE=0, CH_UTF16=0, CH_UNIX, CH_DOS, CH_UTF8, CH_UTF16BE, CH_UTF16MUNGED} charset_t;
```

Reply:
I looked at Samba code and would simply like to keep this change because it is in line with the corresponding Samba code.